libtensor/core/impl/allocator_wrapper.h

```diff
@@ -51,7 +51,7 @@ class allocator_wrapper : public allocator_wrapper_i<T> {
         return ptr.p1;
     }
 
-    virtual void deallocate(const pointer_type &p) throw () {
```

Comment:
here as well

Reply:
I now have performed the redo on those, were there more?

Reply:
No idea. Have you checked with a C++17 compiler?

Reply:
The CI runs, so that is a good sign. I am not sure about the tests. My laptop seems to have been more permanently harmed by the weather, so I need to get that fixed and/or set up my tower pc.